src/assetbundles/src/img/right-arrow.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm1 18v-4h-7v-4h7v-4l6 6-6 6z"/></svg>
```

Comment:
@shnsumit instead of requiring our own SVG, we can just include the `go nowrap` classes on the `<a>` tag like this:

``` html
<a class="go nowrap" href="https://acclarocraft3.com/admin/utilities/updates">Go to Updates</a>
```

That should append the `::after` element.

Reply:
@sidedwards  done,

<img width="1185" alt="image" src="https://user-images.githubusercontent.com/32937137/121419861-129ed100-c98a-11eb-9a7f-95a4134fd0cb.png">